.travis.yml

```diff
@@ -1,10 +1,7 @@
-language: python
-python:
-  - "3.4"
+language: node_js
+node_js:
+  - "5.0.0"
```

Comment:
Actually we haven't thought about it a lot. The main reason of using 5.0.0 here is that we use `jquery` and `jquery-ui` and there are some side effects (that depends on order of imports) that it doesn't work on older versions of Node. There's a plan to introduce native (https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) and after that I believe we will be able to downgrade node to `0.12` or something like that - then we will test it on multiple versions of it.

BTW If You want, You can check in which this engine work feel free to do this (and add it to `.travis.yml`).